portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -5,36 +5,27 @@
 /** Class containing information about a comment left on the site. */
 public class Comment {
   
-  private final String NAME;
-  private final String CONTENT;
-  private int numLikes;
-  private final Date TIMESTAMP;
+  private final String name;
+  private final String content;
+  private long numLikes;
```

Comment:
any reason this is a long instead of int?

Reply:
it's stored as a long in the database but I just changed to cast it to int 081c9b6